docs/src/dev/io/graphson.asciidoc

```diff
@@ -2988,8 +2980,6 @@ The following `ResponseMessage` is a typical example of the typical successful r
 }
 ----
 
-=== Extended
-
 Note that the "extended" types require the addition of the separate `GraphSONXModuleV4d0` module as follows:
```

Comment:
This note needs updating if we are discarding the Core and Extended types categories

Reply:
I decided to leave it as is. GraphSONXModule still exists so it might still be proper to call them "extended" even though those sections in the document have been removed. What do you think?

Reply:
why does "GraphSONXModule" need to exist?

Reply:
It probably doesn't need to exist, but I would use the milestone 1 release to make sure that is the case before updating it.